.circleci/config.yml

```
@@ -184,9 +202,28 @@ workflows:
           requires:
             - build-and-push-apisix-base-dev-arm
             - build-and-push-apisix-base-dev-x86
-  build-and-publish-debian-package:
+  build-mutil-platform-arm64-apckage:
     jobs:
-      - build-and-publish-debian-arm64-package:
+      - build-arm64-package:
+          name: debian-bullseye
+          var_os: debian
+          var_os_release: bullseye-slim
+          filters:
+            tags:
+              only: /^apisix.*/
```

```suggestion
              only: /^apisix\/.*/
```
would be better? So we can filter out apisix-base.

---

No, it also needs to include apisix-base, because it needs to support the packaging of apisix-base